src/target/stm32l0.c

```diff
@@ -372,19 +372,19 @@ static int stm32lx_nvm_prog_erase(target_flash_s *f, target_addr_t addr, size_t
 
```

Comment:
The return just above this can and should be elided and is currently the wrong type